src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -1289,6 +1301,21 @@ private void Audit(IScopeUnitOfWork uow, AuditType type, string message, int use
         /// </summary>
         public static event TypedEventHandler<IMemberService, SaveEventArgs<IMember>> Saved;
 
+        public static event TypedEventHandler<IMemberService, RolesEventArgs> AssignedRoles;
+        public static event TypedEventHandler<IMemberService, RolesEventArgs> RemovedRoles;
+
+        public class RolesEventArgs : EventArgs

```

Comment:
Should go in the Umbraco.Core.Events namespace

Reply:
moving